packages/fuels-programs/src/calls/script_call.rs

```diff
@@ -107,8 +119,12 @@ where
         self
     }
 
-    pub fn with_outputs(mut self, outputs: Vec<Output>) -> Self {
-        self.script_call = self.script_call.with_outputs(outputs);
+    pub fn with_message_outputs(mut self, message_outputs: Vec<Output>) -> Self {
```

Comment:
We agreed to remove this in #795, just a reminder to be careful not to re-introduce it with this PR